CHANGELOG.md

```diff
@@ -13,13 +91,15 @@ This is a minor release with several new features and bug fixes, and is recommen
 - feat: add a field with the formula used to decide to activate stress relief (#805) | [Terra Field](https://github.com/RainOfTerra)
 
 ## Fixes
+
```

Comment:
I thought we successfully turned this off?

Reply:
Weird that it came back again. It should be fixed now